subprojects/docs/src/samples/user-input/publishing-credentials/kotlin/build.gradle.kts

```diff
@@ -0,0 +1,53 @@
+plugins {
+    id("java-library")
+    id("maven-publish")
+}
+
+version = "1.0.2"
+group = "com.example"
+
+// tag::publication[]
+val MAVEN_USERNAME_PROPERTY = "mavenUser"
+val MAVEN_PASSWORD_PROPERTY = "mavenPassword"
+val mavenUser = project.getProviders().gradleProperty(MAVEN_USERNAME_PROPERTY)
```

Comment:
❌ `providers.gradleProperty(...)` would be the idiomatic choice here (instead of `project.getProviders().gradleProperty(...)`)

Reply:
Thanks!
Fixed.